src/nvim/api/command.c

```diff
@@ -40,6 +41,31 @@
 # include "api/command.c.generated.h"
 #endif
 
+/// Parse arguments for mapping commands, preserving whitespace in RHS
+/// @param arg_str  The argument string to parse
+/// @param arena    Arena allocator
+/// @return Array with at most 2 elements: [lhs, rhs]
+static Array parse_mapping_args(const char *arg_str, Arena *arena)
+{
+  Array args = arena_array(arena, 2);  // Allocate for at most 2 arguments
```

Comment:
```suggestion
  MAXSIZE_TEMP_ARRAY(args, 2);
```

Reply:
in any case, the comment isn't needed since that is the express purpose of that arg

Reply:
```suggestion
  Array args = arena_array(arena, 2);
```